.github/workflows/sphinx_docs_to_gh_pages.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: Deploy to GitHub Pages
     runs-on: ubuntu-latest    
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@master
```

Comment:
Might be better to stick to a release version and not the ever changing master branch?